packages/fxa-settings/src/pages/ResetPassword/CompleteResetPassword/index.test.tsx

```diff
@@ -119,16 +132,63 @@ describe('CompleteResetPassword page', () => {
   });
 
   // TODO : check for metrics event when link is expired or damaged
-  it('emits the expected metrics on render when the link is valid', () => {
-    render(
-      <CompleteResetPassword
-        email={MOCK_ACCOUNT.primaryEmail.email}
-        linkStatus="valid"
-      />
-    );
-    expect(usePageViewEvent).toHaveBeenCalledWith(
+  it('emits the expected metrics on render', () => {
+    renderWithAccount({} as unknown as Account);
+
+    expect(logPageViewEvent).toHaveBeenCalledWith(
       'complete-reset-password',
       REACT_ENTRYPOINT
     );
   });
+
+  it('displays any errors', async () => {
+    account = {
+      resetPasswordStatus: jest.fn().mockResolvedValue(true),
+      completeResetPassword: jest
+        .fn()
+        .mockRejectedValue(new Error('Request failed')),
+    } as unknown as Account;
```

Comment:
We have `as unknown as Account` so scattered about in our React codebase, hope we can use those factory functions we talked about in one of those meetings 😅 Maybe we can file an issue for it.